packages/wonder-blocks-core/util/add-style.md

```diff
@@ -30,7 +31,13 @@ prop included ready to be rendered.
 #### StyleType
```

Comment:
One important detail that's missing from this is that `StyleType` can contain a combination of style rules from an Aphrodite StyleSheet as well inline style objects.  It think it'd be good to include the later in one of the examples.

Reply:
That's an excellent point, I'll add that.